exercises/centralLimitTheorem/clm.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+#include <numeric>
+#include <random>
+#include <vector>
+
+std::vector<double> clm(int N, int M) {
+  std::mt19937 gen;
+  std::uniform_real_distribution<double> dis{0., 1.};
+
+  std::vector<double> values(N);
+  for (int i{}; i < N; ++i) {
```

Comment:
Sarebbe bello vedere versioni con range for o con algoritmi. Mettile come soluzioni diverse e commentale nel readme